gulpfile.js

```diff
@@ -29,7 +30,7 @@ function runSpecs(argument) {
     }));
 
   return spec.on('end', function () {
-    process.exit(0);
+    return gulp.src('./coverage/lcov.info').pipe(coveralls());
```

Comment:
@yorkie wow, the bug is hard to find by myself, you are awesome, thank you.
i have correct it on this commit https://github.com/weflex/semver-increment/pull/7/commits/822509f2cac0cb6fd1992a28422202e1db90c89d , please review, thank you.